src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person || {};
+
+  let partnerStatus = '';
+
+  if (isMarried) {
+    if (sex === 'm') {
+      partnerStatus = `${partnerName} is my wife`;
+    } else {
+      partnerStatus = `${partnerName} is my husband`;
+    }
+  } else {
+    partnerStatus = 'I am not married';
+  }
```

Comment:
we can combine it using the ternary operator, try to rewrite it

Reply:
Aha, thanks lot!